### DISCUSSION TITLE
How do I use add_mesh_link?

### DISCUSSION THREAD

**Discussion by jzho2638** [ansys/pyaedt#5309] [General]
I am trying to re-use a mesh from one setup (let's call this Setup1) that has already been solved in another setup (call this Setup2). I am using PyAEDT 0.9.11.

I cannot figure how to use add_mesh_link() using the documentation on the website. Could someone please show me some example code for add_mesh_link() or tell me how I add a mesh-linked solution to a solved/existing Setup? 

Thank you in advance.

**Comment by Samuelopez-ansys** [comment]
Hi @jzho2638 ,

Sometimes the unit tests are a good place to see how a method works:

https://github.com/ansys/pyaedt/blob/main/_unittest/test_20_HFSS.py#L899

This method is used in multiple places (Icepak, Mechanical...) so you will find different usages:
![image](https://github.com/user-attachments/assets/f83c31ba-aaf0-4178-97c0-1e3c237a1761)

I agree, we should add an example in the documentation, please submit an issue if you think it is a good idea.
https://aedt.docs.pyansys.com/version/dev/API/_autosummary/ansys.aedt.core.modules.solve_setup.SetupHFSSAuto.add_mesh_link.html

**Comment by anur7** [comment]
Hi @jzho2638,

I submitted a PR #5341 adding an example in add_mesh_link documentation. Take a look and if you have suggestions please feel free to comment, thanks.
